src/main/java/org/isf/medicalstockward/service/MedicalStockWardIoOperations.java

```diff
@@ -290,5 +290,36 @@ public List<MedicalWard> getMedicalsWardTotalQuantity(char wardId) throws OHServ
 	public long countAllActiveMovementsWard() {
 		return this.movementRepository.countAllActiveMovementsWard();
 	}
+	
+	/**
+	 * Get the {@link MedicalWard} associated to specified criteria.
+	 * @param wardCode the ward code
+	 * @param medical the medical code
+	 * @param lotCode the lot code
```

Comment:
```suggestion
	 * @param lot the lot code
```
The method parameter is `lot` and not `lotCode`